Mathlib/SetTheory/Cardinal/Aleph.lean

```diff
@@ -555,7 +555,8 @@ theorem preBeth_omega : preBeth ω = ℵ₀ := by
 theorem preBeth_pos {o : Ordinal} : 0 < preBeth o ↔ 0 < o := by
   simpa using preBeth_lt_preBeth (o₁ := 0)
 
-theorem isNormal_preBeth : IsNormal (ord ∘ preBeth) := by
+-- TODO: reprove this as `Order.IsNormal preBeth`.
+theorem isNormal_preBeth : Ordinal.IsNormal (ord ∘ preBeth) := by
```

Comment:
Why not just do it now? And since you redefined `Ordinal.IsNormal`, isn't it done?

Reply:
I want to prove that `beth` is a normal function, rather than the composition of `ord` with `beth`. I could do it in this PR, but I figured it'd be nicer to move it to #27015.